object/object_test.go

```diff
@@ -1765,6 +1766,23 @@ func TestObject_Unmarshal(t *testing.T) {
 	require.Equal(t, validObject, obj)
 }
 
+func TestHeaderUnmarshalMarshalCompat(t *testing.T) {
+	// This is a real mwa6Fbia757WvezeeaLfvDsA7hMhdv7bDwN34hBhc97/5vY7ApjJSf9pUynQ33yHRPPXKBE3CQV2HtwfY9Mza4Wa header from mainnet with proto 2.11.
```

Comment:
this is a whole object, not just header, or?

Reply:
i got it, it is an object w/o payload field. A bit different thing from header in API